tests/api/ils/test_mail_permissions.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2021 CERN.
```

Comment:
It would be also nice to add some tests about inserting data in the db and reading it from the endpoint.

There are also some other emails tests: you could assert there that the email log exists in the DB.

Reply:
I have added the tests